proto/provenance/metadata/v1/scope.proto

```diff
@@ -252,3 +253,13 @@ message AuditFields {
   // an optional message associated with the creation/update event
   string message = 6 [(gogoproto.moretags) = "yaml:\"message,omitempty\""];
 }
+
+// NetAssetValue defines a scope's net asset value
+message NetAssetValue {
+  // price is the complete value of the asset's volume
+  cosmos.base.v1beta1.Coin price = 1 [(gogoproto.nullable) = false];
+  // volume is the number of tokens of that the scope was purchased for the price
+  uint64 volume = 2;
```

Comment:
Won't this always be 1? I suppose having this is good for consistency of format.

Reply:
Yes, you are right.  I'm on the fence about removing it now.  Leaving it will keep it identical to the marker's NAV structure, but it isn't serving any purpose and taking up space if it is always 1.

Reply:
I slept on it and I think we should just remove it.  Leaving it just adds something to be managed in the code that should always be 1.  Can cause confusion down the road as well.